status/versions.go

```diff
@@ -207,9 +207,14 @@ type p8sResponseVersion struct {
 }
 
 func jaegerVersion() (*ExternalServiceInfo, error) {
+	jaegerConfig := config.Get().ExternalServices.Tracing
+
+	if !jaegerConfig.Enabled {
+		return nil, nil
+	}
 	product := ExternalServiceInfo{}
 	product.Name = "Jaeger"
-	product.Url = DiscoverJaeger()
+	product.Url = config.Get().ExternalServices.Tracing.URL
```

Comment:
Please replace with `jaegerConfig.URL` as config is read just above (remember that getting config is not without cost as it involves a mutex)